index.html

```diff
@@ -359,12 +359,43 @@ <h3>
       additional members to control audio packetization.
     </p>
     <pre class="idl">partial dictionary RTCRtpEncodingParameters {
-  unsigned long ptime;
-  boolean adaptivePtime = false;
+    RTCResolutionRestriction scaleResolutionDownTo;
+    unsigned long            ptime;
+    boolean                  adaptivePtime = false;
 };</pre>
     <section id="rtcrtpencodingparameters-attributes">
       <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for="RTCRtpEncodingParameters" class="dictionary-members">
+        <dt>
+          <dfn data-idl>scaleResolutionDownTo</dfn> of type <span class="idlMemberType">{{RTCResolutionRestriction}}</span>
+        </dt>
+        <dd>
+          <p>The maximum dimensions at which to restrict this encoding.</p>
+          <p>The default "scale resolution down by" factors (e.g. 4:2:1) are not
```

Comment:
please rephrase without air quotes

Reply:
As per other comment we can remove this sentence because we'll say to ignore it later in the text

Reply:
The issue with the existing `scaleResolutionDownBy` values is that they effectively have default values (inserted in prose even though it's not in WebIDL since the defaults are dynamic, e.g. 4,2,1.

Unfortunately, webcompat here is poor since pasting this into web console produces 4,2,1 values in Firefox, 1,1,1, in Safari, and undefined, undefined, undefined in Chrome as defaults for `scaleResolutionDownBy`: 
```js
new RTCPeerConnection().addTransceiver("video", {sendEncodings: [{rid: "a"}, {rid: "b"}]}).sender.getParameters().encodings;
```

Hopefully this means few websites are actually relying on defaults or they're going to have a hard time.

Long-term we should fix the defaults to be better, but in the short-term we cannot rely on them. Better to ignore them I think

Reply:
This is [crbug 344943229](https://issues.chromium.org/issues/344943229) in Chrome and [webkit 279202](https://bugs.webkit.org/show_bug.cgi?id=279202) in Safari.

Reply:
Done